tests/microbenchmarks.py

```diff
@@ -1,8 +1,8 @@
+from utils import *
```

Comment:
At column 1: (F403) 'from utils import *' used; unable to detect undefined names